docs/docs/deploying-to-clodui.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploying to Clodui
+---
+
+[Clodui] is a static hosting service; Clodui automatically optimizes your website files for speed. This guide explains how to deploy your Gatsby website to Clodui.
+
```

Comment:
Can we add a prerequisite section stating that this guide assumes you have a gatsby site set up and are ready to deploy?

Reply:
thanks for pointing that I'll add it as a prerequisite